website/docs/r/dns_record_set.markdown

```diff
@@ -34,15 +44,25 @@ resource "google_dns_managed_zone" "prod" {
   name     = "prod-zone"
   dns_name = "prod.mydomain.com."
 }
+```
 
-resource "google_dns_record_set" "frontend" {
+### Adding a SPF record
+
+`\"` must be added around your `rrdatas` for a SPF record. Otherwise `rrdatas` string gets splitted on spaces.
```

Comment:
nit: "split" > "splitted"

I think this actually applies to any record that can have spaces in it--TXT is another that comes to mind.